src/main/resources/assets/stylesheets/cvs-version-1.hbs

```diff
@@ -206,7 +206,7 @@ vertical-align: top
 }
 
 .header__three-columns-three {
-width: 163.3333pt;
+width: 183.3333pt;
```

Comment:
I think `.header__three-columns-three` is used on other templates as well. It may impact the formatting on those templates.

Reply:
Good spot, I will leave .header__three-columns-three css as is, and create a custom one to be used specifically by pass and passNoSeatbeltField templates. Will notify once change has been made, tested and re-pushed.